app/javascript/react/components/LocationSearch/LocationSearch.style.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "styled-components";
+
+import { blue, white } from "../../assets/styles/colors";
+import searchIconGray from "../../assets/icons/searchIconGray.svg";
+import { media } from "../../utils/media";
+
+interface SuggestionProps {
+  $isHighlighted?: boolean;
```

Comment:
https://styled-components.com/docs/api#primary
```
"Transient props
If you want to prevent props meant to be consumed by styled components from being passed to the underlying React node or rendered to the DOM element, you can prefix the prop name with a dollar sign ($), turning it into a transient prop."
```